CHANGES.txt

```diff
@@ -1,4 +1,59 @@
-﻿NUnit 3.0.0 Beta 2 - May 12, 2015
+﻿NUnit 3.0.0 Beta 3 - July ??, 2015
+
+Framework
+
+ * The RangeAttribute has been extended to support more data types including 
+   uint, long and ulong
+ * Added platform support for Windows 10 and fixed issues with Windows 8 and
+   8.1 support
+ * Added async support to the portable version of NUnit Framework
+ * Several TestCaseSourceAttribute constructors have been deprecated, see the
+   documentation for more information
+ * RandomAttribute has been extended to add support for new data types including 
+   uint, long, ulong, short, ushort, float, byte and sbyte
```

Comment:
Also TestContext.Random gets the same added support, plus decimal. I'd make that a separate bullet point.